airflow/api_connexion/openapi/v1.yaml

```diff
@@ -1742,7 +1742,7 @@ components:
           nullable: true
           description: Schema of the connection.
         port:
-          type: integer
+          type: string
```

Comment:
Will this fail for the API now?

Reply:
sorry, the wrong file got added.. it was supposed to be the other way of fixing this issue..